client/src/components/FlightMap.vue

```diff
@@ -51,6 +49,12 @@ import { tileOptionsSatellite, tileOptions } from "../config/mapbox";
 import type { FlightTurnpoint } from "@/types/Flight";
 
 import useMapPosition, { type MapPosition } from "@/composables/useMapPosition";
+
+import { leafletMarkerRetinaFix } from "@/helper/leafletRetinaMarkerFix";
+
+// Fix for default marker image paths
+leafletMarkerRetinaFix();
```

Comment:
Würde das dann nicht sogar drei mal ausgeführt? Also jedes mal wenn eine der Komponenten geladen würde. MAcht es da nicht Sinn, dass Ganze zentral, z.B. in der App.vue einmalig auszuführen?

Reply:
Wurde es aktuell ja auch schon. Global bin ich mir nicht sicher, ob das geht. Könnte man mal probieren, aber eher hinten anstellen.